source/ed/connectors/fusio_parser.cpp

```diff
@@ -827,6 +827,11 @@ void CommercialModeFusioHandler::handle_line(Data& data, const csv_row& row, boo
                         "  file has more than one commercial mode and no commercial_mode_id column");
         throw InvalidHeaders(csv.filename);
     }
+    if (! has_col(name_c, row)) {
+        LOG4CPLUS_WARN(logger, "invalid line on " << csv.filename <<
+                        "  commercial mode " << row[id_c] << " is missing a name");
+        return;
+    }
```

Comment:
yes it's strange, a fixture file was containing a windows return line (\r), and with the old gcc I think the vector was empty and is not empty now.

Anyway it makes sens to check the size of the vector before accessing it